observability/network_observability/flowmetrics-api.adoc

```diff
@@ -0,0 +1,9 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="flowmetric-api"]
+= FlowMetrics configuration parameters
+include::_attributes/common-attributes.adoc[]
+:context: network_observability
+
+toc::[]
+
+FlowMetrics is the Schema for the flowmetrics API.
```

Comment:
@jotak Can I mention that you can use the FlowMetric API to create custom metrics here? Just wanting to beef up this section with a little different info.

Reply:
yeah good point, this intro text is really useless at the moment :-)

Reply:
Replaced upstream with:
> FlowMetric is the API allowing to create custom metrics from the collected flow logs.

Is that ok?